app/jobs/deposit_job.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Performs a deposit (without SDR API).
+class DepositJob < ApplicationJob
+  # @param [WorkForm] work_form
+  # @param [Boolean] deposit if true, deposit the work; otherwise, leave as draft
+  def perform(work_form:, deposit:)
```

Comment:
Do we want `deposit` to have a default value, or should we expect callers to provide this arg every time so it's more explicit?

Reply:
Yes, as of now the deposit job will only be invoked in one place (the work controller) and that controller will need to be explicit in determining the deposit value. (It will depend on which button is pressed.)